frontend/src/core/scaffold/scaffoldOverrides.js

```diff
@@ -206,5 +206,4 @@ define(function(require) {
 
 	    return _.isEmpty(errors) ? null : errors;
 	}
-
```

Comment:
I'd avoid changing whitespace in this file since it may cause conflicts with the Backbone Forms upgrade PR.